lk2nd/partial-goods/partial-goods.c

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/* Copyright (c) 2022, Otto Pflüger <affenull2345@gmail.com> */
+
+#include <platform/partial_goods.h>
+#include <boot.h>
+#include <libfdt.h>
+#include <bits.h>
+#include <reg.h>
+
+static void disable_node(void *dtb, const char *path)
```

Comment:
maybe this better to be in util/lkfdt?

Reply:
I've refactored it a bit and moved the helper function, which is now more generic, to util/lkfdt.